src/components/backend-ai-multi-select.ts

```diff
@@ -0,0 +1,262 @@
+/**
+ @license
+ Copyright (c) 2015-2022 Lablup Inc. All rights reserved.
+ */
+
+import {css, CSSResultGroup, html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+import {
+  IronFlex,
+  IronFlexReverse,
+  IronFlexAlignment,
+  IronFlexFactors,
+  IronPositioning
+} from '../plastics/layout/iron-flex-layout-classes';
+import {BackendAiStyles} from './backend-ai-general-styles';
+
+import '@material/mwc-list/mwc-check-list-item';
+import '@material/mwc-icon-button/mwc-icon-button';
+import '@material/mwc-list/mwc-list';
+
+import '../plastics/lablup-shields/lablup-shields';
+
+/**
+ Backend.AI Multi Select
+
+ `backend.ai-multi-select` is multi-selectable UI
+
+ Example:
+
+ <backend-ai-multi-select></backend-ai-multi-select>
+
+ @group Backend.AI Web UI
+ @element backend-ai-multi-select
+ */
+
+@customElement('backend-ai-multi-select')
+export default class BackendAIMultiSelect extends LitElement {
+  public shadowRoot: any; // ShadowRoot
+  @property({type: Object}) comboBox;
+  @property({type: Array}) selectedItemList;
+  @property({type: String, attribute: 'label'}) label = '';
+  @property({type: Array}) items;
+  // TODO: Clear button to remove all selected
+  // TODO: AutoComplete(filtering)
```

Comment:
One more todo (in the future):
Currently, checking one item closes the selection popup. This is inconvenient if a user wants to select/deselect multiple items at once. I think it would be better to keep open the selection popup until a user explicitly closes or the focus is moved to the other elements.

Reply:
Yes, It degrades the user experience. I'll fix it.